### PR TITLE
Add speech icon

### PR DESCRIPTION
**Pull request by AirOne01**
**LEET NICE**

Solves #1329 

Designs available [here](https://www.figma.com/file/cWJVxf9Kc19LcGpANpj5Ah/Lucide?type=design&node-id=0%3A1&t=edUivCN0FdULvIKH-1).

designs

From those, I went with icon 1.

### CODE REVIEW

File: icons/speech.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor"
+  stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+  <path
+    d="M3.5 22 5 16c.4-1.2.2-2-.2-2.9l-.8-1c-.7-.6-1.5-2-1.5-4.1 0-2 1.2-6 6-6 5.3 0 6 3 6 5v5c0 2-1 3-3 3v0a2 2 0 0 0-2 2v5m4.5-3c2.5 0 4.5-2 4.5-4.5m3 0c0 4-3.5 7.5-7.5 7.5" />
+</svg>
```
[the remark this answers is not in the document]
Comment (jguddas) [reply icons/speech.svg:5]: There is also a bit of an issue with the gap being less than 2px, but I have no clue how to fix it :man_shrugging: 

https://github.com/lucide-icons/lucide/assets/25524993/3e688929-502e-4d58-9e18-c937499d5e33

https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit?PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMy41IDIyIDUgMTZjLjQtMS4yLjItMi0uMi0yLjlsLS44LTFjLS43LS42LTEuNS0yLTEuNS00LjEgMC0yIDEuMi02IDYtNiA1LjMgMCA2IDMgNiA1djVjMCAyLTEgMy0zIDNhMiAyIDAgMCAwLTIgMnY1IiAvPgogIDxwYXRoIGQ9Ik0xNCAxOWE0LjUgNC41IDAgMCAwIDQuNS00LjUiIC8+CiAgPHBhdGggZD0iTTIxLjUgMTQuNWMwIDQtMy41IDcuNS03LjUgNy41IiAvPgo8L3N2Zz4=

**Review by jguddas — CHANGES_REQUESTED** [review]

File: icons/speech.svg (hunk 1 above)
[the remark this answers is not in the document]
Comment (AirOne01) [reply icons/speech.svg:5]: I had no idea as well so I just made the PR anyways. My second icon is a failed attempt at fixing this 🤷‍♂️

[the remark this answers is not in the document]
Comment (AirOne01) [reply icons/speech.svg:5]: Made two new versions:

| |
| --- |
| image |
| image |

I think no. 4 looks more impactful but I'd stay on number 3 (closes more on the feeling of feather). Making my changes.

**Comment by AirOne01** [comment]
I also used the tool you link to make proper SVG this time. Couldn't find a way (pretty) to make this padding exactly 2px though.

**Comment by jguddas** [comment]
Mr. cool wearing a hat icon (82)

**Comment by karsa-mistmere** [comment]
Nice concept, although I feel like the head shape might be a bit too blobby, how about [something a bit more geometric](https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit?PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJtMiAxOSAuMy0uOGEyNi4wNyAyNi4wNyAwIDAgMCAxLjAzMS00LjUxNyAzLjU1IDMuNTUgMCAwIDAtLjU1Mi0yLjEyNkE3LjMzIDcuMzMgMCAwIDEgMiA4YTYgNiAwIDEgMSAxMiAwdjNhMyAzIDAgMCAxLTMgM2gtMWExIDEgMCAwIDAtMSAxdjQiIC8+CiAgPHBhdGggZD0iTTEzIDE4YTUgNSAwIDAgMCA1LTUiIC8+CiAgPHBhdGggZD0iTTIyIDEzYzAgNC43NjUtNC4yMzUgOS05IDkiIC8+Cjwvc3ZnPg==)?

**Comment by karsa-mistmere** [comment]
Another alt w/ maybe more traditionally placed sound waves:
speech
https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit?PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNOC44IDIwdi00LjFsMS45LjJhMi4zIDIuMyAwIDAgMCAyLjE2NC0yLjFWOC4zQTUuMzcgNS4zNyAwIDAgMCAyIDguMjVjMCAyLjguNjU2IDMuMDU0IDEgNC41NWE1Ljc3IDUuNzcgMCAwIDEgLjAyOSAyLjc1OEwyIDIwIiAvPgogIDxwYXRoIGQ9Ik0xOS44IDE3LjhhNy41IDcuNSAwIDAgMCAuMDAzLTEwLjYwMyIgLz4KICA8cGF0aCBkPSJNMTcgMTVhMy41IDMuNSAwIDAgMC0uMDI1LTQuOTc1IiAvPgo8L3N2Zz4=

**Comment by AirOne01** [comment]
This is way better, thanks! I'll make my changes when I'm off work.

**Pull request merged**